### PR TITLE
Add LEECH_ONLY flag and logic

### PR DESCRIPTION
**Pull request by QR4SH3D**
The bot was wasting a lot of time when trying to kill invoke without Energy Leech

### CODE REVIEW

**Comment by pajtor** [comment]
Did you at least test it before making pull request?

**Comment by ODAKAB** [comment]
> Did you at least test it before making pull request?

It is tested, I cooperated with him.

**Comment by pajtor** [comment]
> > Did you at least test it before making pull request?
> 
> It is tested, I cooperated with him.

image

File: src/main/resources/lang/strings_en.properties [hunk 1]
```diff
@@ -147,6 +147,9 @@ config.loot.npc_table.extra.passive.desc=Be passive towards this NPC, only shoot
 config.loot.npc_table.extra.attack_second=Attack second
 config.loot.npc_table.extra.attack_second.short=AS
 config.loot.npc_table.extra.attack_second.desc=<html>Only shoot if others are attacking already.<br><strong>Must</strong> also select ignore attacked & ignore ownership</html>
+config.loot.npc_table.extra.leech_only=Leech only
+config.loot.npc_table.extra.leech_only.short=LO
+config.loot.npc_table.extra.leech_only.desc=Attack under Energy Leech effect
```
Comment (Pablete1234) [inline src/main/resources/lang/strings_en.properties:152]:
```suggestion
config.loot.npc_table.extra.leech_only.desc=Only attack when energy leech effect is active
```

Comment (QR4SH3D) [reply src/main/resources/lang/strings_en.properties]: Done 👍🏼

**Comment by QR4SH3D** [comment]
> Did you at least test it before making pull request?
> 
> 

Make sure you test your plugin too 😐 and dont point your finger on others 😉

**Comment by pajtor** [comment]
> > Did you at least test it before making pull request?
> > 
> > 
> 
> Make sure you test your plugin too 😐 and dont point your finger on others 😉

I at least code mostly by myself and not screensharing and doing what somebody tells me lmao

**Comment by ODAKAB** [comment]
Why you guys are doing dogfight under a pull request?

**Comment by pajtor** [comment]
> Why you guys are doing dogfight under a pull request?

To spam git-log

**Comment by Punikekk** [comment]
<:pepebox:756138542953922642>

**Pull request merged**